Frameworks/include/CGContextCairo.h

```diff
@@ -138,7 +138,7 @@ class CGContextCairo : public CGContextImpl {
     virtual bool CGContextIsPointInPath(bool eoFill, float x, float y);
     virtual CGPathRef CGContextCopyPath(void);
 
-    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun);
+    virtual void CGContextDrawGlyphRun(const DWRITE_GLYPH_RUN* glyphRun, float lineHeight);
 
```

Comment:
I'm concerned that this gives the caller too much control (or, alternatively, _the need_ for too much control)